jenkins/run

```diff
@@ -67,26 +85,52 @@ _image=${IMAGE:-${_image_repo}/pbench-${_image_role}-${_image_kind}:${_branch_na
 if [ -n "${CONTAINER_HOST}" ]; then
     # Since CONTAINER_HOST is defined, when we want to run another container, do
     # it with `podman-remote` (which will run it where CONTAINER_HOST points).
-    # This works nicely, e.g., if we are already inside a properly configured
-    # container.
+    # This is the core mechanism that allows CI jobs, which are run inside one
+    # container, to run sub-sets of the build inside separate containers on the
+    # same host.
     PODMAN="podman-remote"
 else
-    # Since CONTAINER_HOST is not defined, don't invoke the container remotely.
-    # However, we might want to run a container "remotely" on this host;
-    # therefore, enable the service which makes it available to receive
-    # `podman-remote` invocations.  This creates a listener on
-    # /run/user/$(id -u)/podman/podman.sock, which will be mapped into the
-    # container in the invocation below so that we can create peer-containers
-    # on this host from inside the container.
+    # Since CONTAINER_HOST is not defined, arrange to invoke the container
+    # directly.  Since the code running in the container might want to run an
+    # additional container "remotely" on this host, enable the service which
+    # makes it available to receive `podman-remote` invocations.  This creates a
+    # listener on /run/user/$(id -u)/podman/podman.sock, which will be mapped
+    # into the container in the invocation below so that we can create
+    # peer-containers on this host from inside the container.
     PODMAN="podman"
     systemctl --user enable --quiet --now podman.socket
 fi
 
+# We have two directory trees which we are concerned with inside the container:
+# $HOME_DIR and $HOME_DIR/pbench.  The first is provided for anything which
+# happens to reference it (such as RPM builds and npm), and the second contains
+# the Git checkout.
+#
+# When this script is run under the CI (or in any other environment which
```

Comment:
In fact, given this and the subsequent paragraph, I think the first paragraph probably adds nothing and could simply be removed.

Reply:
The principal thing that the first paragraph does is point out that there are things in the build (to wit, `npm` and the RPM builds) which rely on the `$HOME` directory (this was a surprise to me, initially, on each count...), and it explains why we bother to set up a mapping for it.  The other thing that it does is underscore the fact that the home directory is handled separately from the Git checkout directory (but, as you say, this is described further in the subsequent text).